Find a functionally equivalent code:android.os.CountDownTimer.cancel:COMMENT
Method Modifier: public      final       
Comment:/**
 * Cancel the countdown.
 */

@@ -1,3 +1,4 @@
 {
+    mCancelled = true;
     mHandler.removeMessages(MSG);
 }

